pkg/pushsync/pushsync_test.go

```diff
@@ -34,25 +36,27 @@ const (
 )
 
 // TestSendChunkAndGetReceipt inserts a chunk as uploaded chunk in db. This triggers sending a chunk to the closest node
-// and expects a receipt. The message are intercepted in the outgoing stream to check for correctness.
+// and expects a receipt. The message are intercepted in the outgoing stream to check for correctness.cry
```

Comment:
The test should not make anyone cry :) at least in an ideal world.

Reply:
I have no idea how that ended up there :)